File: infra/src/main/java/whenyourcar/infra/objectStorage/properties/ObjectStorageProperties.java
package whenyourcar.infra.objectStorage.properties;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Getter
@Setter
@Component
@ConfigurationProperties(prefix = "spring.oci.object-storage")
public class ObjectStorageProperties {
    private String region;
    private String namespace;
    private String bucket;
}

File: infra/src/main/kotlin/howmuchcar/infra/properties/ObjectStorageProperties.kt
package howmuchcar.infra.properties

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component


@Component
@ConfigurationProperties(prefix = "spring.object-storage")
data class ObjectStorageProperties(
    var region: String? = null,
    var namespace: String? = null,
    var bucket: String? = null
)